code/src/Albums.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { Icons } from "./Icons";
+
+export const Albums = props => {
+  return (
+    <div
+      className="AlbumContainer"
+      style={{ backgroundImage: `url(${props.image})` }}
+    >
+      <div className="ArtistContainer">
+        <div>
+          <div className="iconsContainer">
+            <Icons />
+          </div>
+          <div className="ArtistContent">
+            <a className="titleLink" href={props.albumURL}>
+              <p className="AlbumName">{props.name}</p>
+            </a>
```

Comment:
I think the `<p>` should be outside the `<a>` tag here (instead of the other way around) since it makes the link become a block which means if you hover by the side of the artist name, it hovers the artist name - if that makes sense?

It's not _wrong_ to do it this way, but I think the other way would be nicer in this case.